plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/GlobalConfigurationCategoryConfigurator.java

```diff
@@ -69,13 +69,14 @@ protected GlobalConfigurationCategory instance(Mapping mapping, ConfigurationCon
         return category;
     }
 
+    @SuppressWarnings("RedundantCast") // TODO remove once we are on JDK 11
```

Comment:
weird, what does it think it is?
you would think 
```
.collect(Collectors.toSet());
```

would get you a set?

Reply:
@timja 
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/travis/build/jenkinsci/configuration-as-code-plugin/plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/GlobalConfigurationCategoryConfigurator.java:[81,25] incompatible types: java.lang.Object cannot be converted to java.util.Set
```

https://travis-ci.org/jenkinsci/configuration-as-code-plugin/jobs/513786750#L659